.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: UI Tests
+
+on:
+  workflow_dispatch:
+  deployment_status:
+
+env:
+  GIT_PAT: ${{ secrets.GIT_PAT }}
+  MNEMONIC: ${{ secrets.MNEMONIC }}
+  MNEMONIC_MAINNET: ${{ secrets.MNEMONIC_MAINNET }}
+  PATH_COSMOSTATION: './wallets/cosmostation'
+  PATH_KEPLR: './wallets/keplr'
+  PATH_LEAP: './wallets/leap'
+  PATH_METAMASK: './wallets/metamask' 
+  WALLET_PASSWORD: ${{ secrets.WALLET_PASSWORD }}
+
+jobs:
+  github-actions-testnet:
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
```

Comment:
How do we make a successful `deployment_status` event?

I have a deploy preview process in workflows, but I don't think it uses these events? 
https://github.com/duality-labs/duality-web-app/blob/v2.2.1/.github/workflows/deploy_preview.yml#L130-L131

Reply:
Noting that this is fixed with commits:
- e84fc916753e7c3683eb92c95046cf43362d9491
- 94807117742ec791df1572fab4c03cf6094ead9b

That add a [GitHub deployment event](https://docs.github.com/en/rest/deployments/deployments?apiVersion=2022-11-28#create-a-deployment) (`github.event.deployment_status`) using a `chrnorm/deployment-action@v2` action